### PR TITLE
add Docker image upload trigger method and add docker image labeling

### PR DESCRIPTION
**Pull request by Seochokid**
- add Docker image upload trigger method (push to main branch)
- add docker image labeling (to link container registry repo)

### CODE REVIEW

File: src/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Start from latest GO base image
 FROM ubuntu:18.04
+LABEL org.opencontainers.image.source https://github.com/poseidonos/poseidonos-gui
```
Comment (so931) [inline src/Dockerfile:3]: Is it possible to explain the purpose of this label?

Comment (Seochokid) [reply src/Dockerfile:3]: With this label, container images are linked to this repo (poseidonos-gui) registry and displayed.
If not,  container images are linked to poseidonos org.

Comment (so931) [reply src/Dockerfile:3]: Thank you for the explanation.

**Review by so931 — APPROVED** [review]
lgtm

**Review by aswinssir — APPROVED** [review]
LGTM

**Pull request merged**